internal/service/service_integration/resource_service_integration.go

```diff
@@ -148,7 +148,7 @@ func ResourceServiceIntegration() *schema.Resource {
 			StateContext: schema.ImportStatePassthroughContext,
 		},
 		Timeouts: &schema.ResourceTimeout{
-			Create: schema.DefaultTimeout(5 * time.Minute),
+			Create: schema.DefaultTimeout(10 * time.Minute),
```

Comment:
Customers can configure this field for themselves using something like: 
```
resource "aiven_service_integration" "test" {
  ....
  timeout {
    create = 10m
  }
}
```

Reply:
Yeah, I know. But _default_ is not enough.